services/QuillLMS/client/package.json

```diff
@@ -76,6 +76,7 @@
     "react-css-transition-replace": "^4.0.2",
     "react-csv": "^1.0.8",
     "react-dates": "21.8.0",
+    "react-datetime-picker": "^3.3.0",
```

Comment:
I feel like we shouldn't be adding another datepicker library when we already have `react-dates` in here. Is there a reason `react-dates` doesn't work for what we need?

Reply:
@emilia-friedberg yeah so in the spec, it's specified that they want to be able to select both the date and time. `react-dates` is just a calendar widget and only let's you specify the date